lib/perl/Genome/Model/ClinSeq/Command/AnnotateSnvsVcf/Result.pm

```diff
@@ -0,0 +1,72 @@
+package Genome::Model::ClinSeq::Command::AnnotateSnvsVcf::Result;
+
+use strict;
+use warnings;
+use Genome;
+use File::Basename qw(fileparse);
+
+class Genome::Model::ClinSeq::Command::AnnotateSnvsVcf::Result {
+    is => 'Genome::SoftwareResult::StageableSimple::SingleFile',
+    has_input => [
+        input_file => {
+            is => 'Text',
+            doc => 'Vcf File to filter',
+        },
+        annotation_file => {
+            is => 'Text',
+            doc => 'Vcf File containing annotation',
+        },
+        info_fields => {
+            is => 'Text',
+            doc => 'Field ids to embed from the annotation VCF. Use colons to separate multiple field descriptors.',
+            #doing the above because UR autosplits on commas with is_many, but joinx uses commas in its field descriptors
+        },
+    ],
+    has_optional_input => [
+        identifiers => {
+            is => 'Boolean',
+            doc => 'copy identifiers from the annotation file',
+        },
+        info => {
+            is => 'Boolean',
+            doc => 'copy information from info fields from the annotation file',
+        },
+    ],
+};
+
+sub _run {
+    my $self = shift;
+
+    my $vcf_annotator = Genome::Model::Tools::Joinx::VcfAnnotate->create(
+        input_file => $self->input_file,
+        annotation_file => $self->annotation_file,
+        output_file => $self->_temp_staging_file_path,
+        use_bgzip => $self->use_bgzip,
+        info_fields => $self->info_fields || "",
```

Comment:
Is it okay that this would normalize `0` to the empty string?  (Looks like `info_fields` is a required property; is this trying to catch a particular issue that has happened?)